jenkins/Jenkinsfile-release

```diff
@@ -17,7 +17,8 @@ IMAGE = "${REGISTRY}/opsh2oai/h2o-flow-build"
 MASTER_BRANCH = 'master'
 REL_BRANCH_PREFIX = 'rel-'
 
-def flowVersion = "0.8.${currentBuild.number}"
+def FLOW_VERSION_MAJOR = '0.8'
```

Comment:
would it be better to have this defined in a properties file in the repository, not in the pipeline itself?